### PR TITLE
[Outlook] Note max on props that return Recipients object

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fa973c0](https://github.com/OfficeDev/office-js-docs-pr/commits/fa973c0688912779fa8488dd969752be545c0f24)_: 

:clock10: **Pending**: waiting for processors ([5](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 5 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fa973c0](https://github.com/OfficeDev/office-js-docs-pr/commits/fa973c0688912779fa8488dd969752be545c0f24)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910142044003426-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ojWv7mNi6LUMXgwuDUl815byqPov%2FSt%2BNb38oPJinz0%3D&st=2019-10-14T20%3A40%3A19Z&se=2019-11-14T20%3A45%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1dffc4a](https://github.com/OfficeDev/office-js-docs-pr/commits/1dffc4a599a7fb9e129dc7ea6a7e0af9b94c3302)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910182132079614-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7W%2B%2BuM4E5wd8i0NuM5kBR8wyI%2FmA9n6R9x3edJOzuFU%3D&st=2019-10-18T21%3A28%3A20Z&se=2019-11-18T21%3A33%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fa973c0](https://github.com/OfficeDev/office-js-docs-pr/commits/fa973c0688912779fa8488dd969752be545c0f24)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910182132182889-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=d%2Bq9QDreJiNr%2F1RKxwflfV4EzQL7V8L7JZtRe%2FJ3oP8%3D&st=2019-10-18T21%3A29%3A53Z&se=2019-11-18T21%3A34%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b9d0b84](https://github.com/OfficeDev/office-js-docs-pr/commits/b9d0b848a566aa5c5d61fd643c995ce7bf622b19)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910182135597950-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=PKRQPYkMl%2FSGce%2B0WovHrdyLW369W1CgPgNy8jDih3w%3D&st=2019-10-18T21%3A32%3A25Z&se=2019-11-18T21%3A37%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md [hunk 1]
```diff
@@ -149,6 +149,8 @@ console.log(outputString);
 
 Gets an object that provides methods to get or update the recipients on the Bcc (blind carbon copy) line of a message. Compose mode only.
 
+By default, the collection is limited to a maximum of 100 members. However, on Windows and Mac, you can set up to 100 members per call to 500 total, and get 500 members maximum.
```
Comment (Rick-Kirkham) [inline docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md]: The "However, " sentence is puzzling. is it supposed to mean "**up** to 500 total"? Also,, why is the 500 maximum being specified twice?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[00f9a69](https://github.com/OfficeDev/office-js-docs-pr/commits/00f9a695c74b38cf413e3a698a89d95b6c84a43c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910232224389925-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZyO4xFHH9nXqHa8GD48iMGnPw3JBYjvkKCIc%2F8pkvvQ%3D&st=2019-10-23T22%3A21%3A04Z&se=2019-11-23T22%3A26%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fe1f21d](https://github.com/OfficeDev/office-js-docs-pr/commits/fe1f21d0794eec38256f01b1874c4ab4c438f952)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-recipients-max/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910232318048136-1363%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=YYsayoUnlu61P2dQAzpYaWvmfeQNHWea5Q4uqrhFFfI%3D&st=2019-10-23T23%3A14%3A26Z&se=2019-11-23T23%3A19%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**